front-ps/src/Guide/View/CategoryDetail.purs

```diff
@@ -8,16 +8,16 @@ import Guide.Events (Event(..))
 import Guide.Routes (Route(..), toUrl)
 import Guide.State (State(..))
 import Guide.Types (CategoryName(..))
-import Guide.Utils (Uid(..))
+import Guide.Api.ClientTypes (CUid(..))
 import Pux.DOM.Events (onClick) as P
 import Pux.DOM.HTML (HTML) as P
 import Text.Smolder.HTML (div, h1, a) as S
 import Text.Smolder.HTML.Attributes (href) as S
 import Text.Smolder.Markup ((#!), (!))
 import Text.Smolder.Markup (text) as S
 
-view :: CategoryName -> Uid String -> State -> P.HTML Event
-view (CategoryName cName) (Uid catId) (State st) =
+view :: CategoryName -> CUid String -> State -> P.HTML Event
+view (CategoryName cName) (CUid catId) (State st) =
   S.div $ do
     S.h1 $ S.text (cName <> " - " <> catId.uidToText)
```

Comment:
Would be nice just to use `catId` instead of `catId.uidToText` (see my other comments)